pages/formester-vs-paperform.vue

```diff
@@ -211,7 +211,7 @@
       </div>
     </section>
 
-    <section class="container text-center">
+    <section class="container text-center mb-5">
```

Comment:
Is this `container` class from bootstrap If not check if we can add `margin-bottom` in that class so we don't need to use `mb-5` every time, If it is from bootstrap then it is fine

Reply:
yes @hanzalakhan809 container is a bootstrap class, so I have added mb-5 class separately.